web/ui/src/Pools/PoolsController.js

```diff
@@ -6,15 +6,38 @@
 (function() {
     'use strict';
 
-    controlplane.controller("PoolsController", ["$scope", "$routeParams", "$location", "$filter",
-        "$timeout", "resourcesFactory", "authService", "$modalService", "$translate", "$notification",
-        "miscUtils", "poolsFactory", "areUIReady",
-    function($scope, $routeParams, $location, $filter, $timeout,
-    resourcesFactory, authService, $modalService, $translate, $notification,
-    utils, poolsFactory, areUIReady){
+    // Pool object constructor takes a pool object (backend pool object)
+    // and wraps it with extra functionality and info
+    function Pool(pool){
+        this.id = pool.ID;
+        this.model = Object.freeze(pool);
+    }
+
+    controlplane.controller("PoolsController", ["$scope", "$routeParams",
+        "resourcesFactory", "authService", "$modalService", "$translate",
+        "$notification", "areUIReady", "$interval", "servicedConfig", "log",
+    function($scope, $routeParams, resourcesFactory, authService, $modalService,
+             $translate, $notification, areUIReady, $interval, servicedConfig, log){
+
         // Ensure logged in
         authService.checkLogin($scope);
 
+        var lastUpdate;
```

Comment:
Can you group all of the update related stuff together, and put a comment indicating that its the remnants of poolsfactory? This way itll be easier to identify and remove or factor our in the future.